containers/python/rpc.py

```diff
@@ -488,4 +503,37 @@ def start(self, model, host, port, model_name, model_version, input_type):
         self.server.model_version = model_version
         self.server.model_input_type = model_input_type
         self.server.model = model
-        self.server.run()
+
+        parent_conn, child_conn = Pipe(duplex=True)
+        metrics_proc = Process(target=run_metric, args=(child_conn, ))
+        metrics_proc.start()
+        self.server.run(parent_conn)
+
+
+class MetricCollector:
+    def __init__(self, pipe_child_conn):
+        self.pipe_conn = pipe_child_conn
+
+    def collect(self):
+        curr = None
+        while self.pipe_conn.poll():
+            curr = self.pipe_conn.recv()
```

Comment:
The purpose is to just get the the last item. Prometheus 'scrape' for the latest metric every 5 seconds and the metric is updated only for each `GET` call. 

Because the prediction time is variable, in the rpc loop, we push the most recent metric dictionary into the pipe after each prediction call. In the metric collector, we just need to get the most recent one. 

I changed the name to `lastest_metric_dict` for better clarity.